include/metal/lambda/arg.hpp

```diff
@@ -19,7 +19,7 @@ namespace metal {
     /// A parametric \lambda that selects the n-th argument it is invoked with.
     ///
     /// ### Usage
-    /// For any non negative integral value `m`
+    /// For any non zero positive integral value `m`
```

Comment:
I suggest to shorten this line to 

> For any positive integral value `m`

If you insist on being explicit here, then I prefer "nonzero" over "non zero".

Reply:
I like your rigor :)